addon/components/x-select.js

```diff
@@ -77,7 +77,24 @@ export default Ember.Component.extend({
       this._updateValueSingle();
     }
 
-    this.sendAction('action', this.get('value'), this);
+    this.sendAction('action', this.get('value'), this); // should we deprecate `action` in favor of `on-change`?
+    this.sendAction('on-change', this.get('value'), this);
```

Comment:
Since we're adding other event actions I feel like we should deprecate the `action` action in favor of a more descriptive name like `on-change`.